charts/devlake/values.yaml

```diff
@@ -200,6 +204,8 @@ lake:
 
   containerSecurityContext: {}
 
+  annotations: {}
+
```

Comment:
kindly change annotations to podAnnotations

Reply:
updated to `podAnnotations` as requested